canonical/canonical.com
# Simplify the navigation across the site

## Done
- Removed the navigation dropdowns
- Made the navigation consistent across the site
- Improved the secondary styling in the partners section
- Removed tons of code

## QA
- Use the navigation and see its more consistent
- Go to /partners and see the secondary nav looks better but the same

## Issue / Card
Fixes https://github.com/canonical-web-and-design/canonical.com/issues/124
